Helper/Data.php

```diff
@@ -838,4 +839,32 @@ public function allowMinicartCheckout($store = \Magento\Store\Model\ScopeInterfa
     {
         return (bool)$this->scopeConfig->getValue(self::XML_CONGIG_MINICART_CHECKOUT, $store);
     }
+
+    /**
+     * Get threshold amount below which if set bread button or checkout won't be enabled
+     *
+     * @param string $store
+     * @return int
+     */
+    public function getThresholdAmount($store = \Magento\Store\Model\ScopeInterface::SCOPE_STORE)
```

Comment:
I don't think we need a whole other function for just this one line getValue call, I think it'll be clear enough what it's doing to just have that one line in the aboveThreshold function

Reply:
Changed